examples/shopping/get_product_bidding_category_constant.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
```

Comment:
On the first line please include the shebang, license and also a top-level comment describing what this example does. See [here](https://github.com/googleads/google-ads-python/blob/master/examples/advanced_operations/add_gmail_ad.py#L1) for an example.